txtdirect_test.go

```diff
@@ -407,6 +409,48 @@ func RunDNSServer() {
 	err := server.ListenAndServe()
 	defer server.Shutdown()
 	if err != nil {
-		log.Fatalf("Failed to start server: %s\n ", err.Error())
+		log.Printf("Failed to start server: %s\n ", err.Error())
+	}
+}
+
+func TestRedirectTypeHost(t *testing.T) {
```

Comment:
Not sure the function name reflects the actual tests being run.

Reply:
Oops! forgot to rename it, my first idea was to write single tests for each type :rofl:

Reply:
TestRedirectE2e?